.github/workflows/fuzzing.yml

```
@@ -152,5 +152,79 @@ jobs:
+    - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+      with:
+        submodules: 'recursive'
+        ref: fuzz/corpus
+
+    - name: Download fuzzer artifacts
+      uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16
+      with:
+        name: fuzzer-${{ matrix.platform }}-${{ matrix.arch }}
+
+    - name: Setup directory for fuzzing
+      run: |
```

actionlint

shellcheck reported issue in this script: SC1001:info:3:10: This \c will be a regular 'c' in this context

(shellcheck)